lldb/include/lldb/Utility/CompletionRequest.h

```diff
@@ -104,7 +104,7 @@ class CompletionRequest {
 
   llvm::StringRef GetRawLine() const { return m_command; }
   llvm::StringRef GetRawLineUntilCursor() const {
-    return m_command.substr(0, m_cursor_index);
+    return m_command.substr(0, m_raw_cursor_pos);
```

Comment:
Do we have a way to test this change? Is it tested as part of your other commits?

Reply:
Two of the commits here are not Swift-specific so I'll make a separate PR for them to the stable branch once this passes the test.

Reply:
Whoops, somehow my commented ended up in this thread.... I jus pushed a unit test for this change too.